bootstrap.sh

```diff
@@ -1,10 +1,19 @@
 #!/bin/sh
 
+set -e
+
 echo Building the build tool...
 
 GOBIN="$(pwd)/util" go get -u harvey-os.org/cmd/ufs
 GO111MODULE=on GOBIN="$(pwd)/util" go get ./util/src/harvey/cmd/...
 
+echo Fetching u-root and building it...
+GO111MODULE=off go get github.com/u-root/u-root
+GO111MODULE=off go install github.com/u-root/u-root
```

Comment:
Oh I will need to keep that in mind, might overwrite other experiments... that installs to `$GOPATH/bin`, so usually `$HOME/go/bin`, right? Could we keep it local here?

Reply:
Shall we stick with util/ then? 

The one good thing about util/, as opposed to bin/ at the top level, is that nobody will think util/ has harvey binaries in it. 

OTOH ... what about a top level directory called $GOOS_$GOARCH, with a bin/ in it, so we can correctly name binaries?

Reply:
I like that composition, it's similar to what u-root uses for naming cpio files. 👍

Reply:
oops, I thumbed up my own PR